### ISSUE TITLE
Genome List: number of records per page / add to gorup

### ISSUE THREAD

**Issue by mshukla1** [PATRIC3/patric3_website#791]
As of now, max 200 genomes are displayed per page and there is no way to change that number. 

It is  cumbersome to add more 1000 or more genomes to a group, as it will require user to 
go through multiple pages. 

We need 
- Ability to increase the number of records displayed per page
- True "Select all" results button, so that all genomes/genes from a search result can be added to a group in a single click. 

-Maulik

**Comment by dmachi** [comment]
I've added in the ability to select the number of records displayed per page (still needs to be styled). I don't know that I agree with doing this, but i've added it in for now.  I think the better option is to get the select all working better so this isn't necessary.

**Comment by dmachi** [comment]
I've updated selectAll to get all data up to  a maximum of 5000 records. If a table contains more than this, only the first 5000 records will be selected.  This should avoid the group problem you mentioned without allowing millions of records to be selected.  Ultimately I think Groups should be capped at around 10k items.  Since this feature is available, I've removed the pages selector.  Will be pushed to alpha soon.  In some cases select all may be impossible and we'll need to disable it there.  Additionally, I'm still testing/updating all of the actions.  Some actions may still fail (specifically some download buttons) if the selection comes from the use of the selectAll button.
